library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -312,6 +312,15 @@ public int getItemCountForGroup(int groupIndex) {
         return groups.get(groupIndex).getItemCount();
     }
 
+    /**
+     * This returns the total number of items contained in the top level group at the passed index
+     * @deprecated This method has been deprecated in favour of {@link #getItemCountForGroup(int)}. Please use that method instead.
```

Comment:
It would have been awesome if we had this file in Kotlin - we could have used the `@ReplaceWith` annotation like described here: https://hackernoon.com/how-kotlins-deprecated-relieves-pain-of-colossal-refactoring-8577545aaed.

This should work for now though!

Reply:
Yes it would have indeed. I have explored converting the library to Kotlin but the benefit of a direct conversion isn't that great over just keeping it in Java. It'll be worth it more if we decide to do a larger API overhaul that leverages Kotlin features a lot more.